src/control/cmd/daos/pretty/pool.go

```diff
@@ -63,6 +65,42 @@ func printPoolTiersMdOnSsd(memFileBytes uint64, suss []*daos.StorageUsageStats,
 	}
 }
 
+// PrintSelfHealPolicyDisable compares system and pool self_heal flags and display disabled
+// features.
+func PrintSelfHealPolicyDisable(poolSelfHeal, sysSelfHeal string, out io.Writer) {
```

Comment:
[Nit] Would `PrintPoolSelfHealPolicyDisable` or `PrintPoolSelfHealDisable` (i.e., emphasizing "pool") be more accurate?

Reply:
fixed, done